iree/base/wait_handle_impl.h

```diff
@@ -52,7 +52,13 @@
 // TODO(benvanik): EPOLL on android/linux/bsd/etc.
 // TODO(benvanik): KQUEUE on mac/ios.
 #if !defined(OS_IOS) && !defined(__EMSCRIPTEN__)
+
+#if defined(IREE_PLATFORM_MACOS)
+#define IREE_WAIT_API IREE_WAIT_API_KQUEUE
```

Comment:
The kqueue implementation isn't done yet. You should be able to set IREE_WAIT_API_POLL instead by replacing the OS_IOS check above with `!defined(IREE_TARGET_PLATFORM_APPLE)`.

Reply:
(kqueue is much more efficient than poll and we'll want to implement it at some point :)

Reply:
Got it. Thanks